tool/get_localid

```diff
@@ -1,100 +1,100 @@
-#!/usr/bin/python
```

Comment:
Did you just rename the files?  Please don't, I will do that in a soon upcoming patch (if fact, this is going to be C code really soon because it does not work well with the database from Python).